projects/Amlogic/options

```diff
@@ -9,7 +9,7 @@
     UBOOT_FIRMWARE="amlogic-boot-fip"
 
   # Linux Kernel to use
-    LINUX="default"
+    LINUX="amlogic-5.2"
```

Comment:
```suggestion
    LINUX="amlogic"
```

Reply:
it adds a tiny complication to the merged branch but helps keep things simple when you're hopping between 5.1/5.2/5.3 branches to test things.